caff-node/config/caff_node.json

```diff
@@ -0,0 +1,81 @@
+{
+    "parent-chain": {},
+    "chain": {
+        "id": 1000000,
+        "info-files": [
+            "/config/l2_chain_info.json"
+        ]
+    },
+    "node": {
+        "staker": {
+            "enable": false
+        },
+        "sequencer": true,
+        "dangerous": {
+            "no-sequencer-coordinator": true,
+            "disable-blob-reader": true,
+            "no-l1-listener": true
+        },
+        "delayed-sequencer": {
+            "enable": false
+        },
+        "parent-chain-reader": {
+            "enable": false
+        },
+        "seq-coordinator": {
+            "enable": false
+        },
+        "batch-poster": {
+            "enable": false
+        },
+        "block-validator": {
+            "enable": false
+        },
+        "feed": {
+            "input": {},
+            "output": {
+                "enable": false,
+                "signed": false,
+                "addr": "0.0.0.0"
+            }
+        },
+        "data-availability": {
+            "enable": false
+        }
+    },
+    "execution": {
+        "sequencer": {
+            "enable": true,
```

Comment:
I'm not sure which version of the caff node you are working against, but we are making it so that `sequencer.enable` will be set to false here, and on line 11 for the caff node in the future, but if we are targeting an older version then that's fine.

Reply:
I'm using the earlier version Sneh shared with me that is compatible with ARM architecture.  Would that be okey?